udata/core/organization/api.py

```diff
@@ -207,10 +210,6 @@ def delete(self, org, badge_kind):
         return badges_api.remove(org, badge_kind)
 
 
-from udata.core.contact_point.api import ContactPointApiParser
```

Comment:
I  had in mind that @ThibaudDauce put the import here for circular imports reason, but it may not be the case anymore

Reply:
Hmm I guess it's okay with what you've described